### PR TITLE
Optimize heartbeat scheduling loop

### PR DESCRIPTION
**Pull request by mayurva**
sched.enter needs to be called only once for scheduling the heartbeat loop

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mayurva has signed the CLA. Thanks for the pull request!

File: app/master/cluster_master.py [hunk 1]
```diff
@@ -70,12 +70,13 @@ def start_heartbeat_tracker_thread(self):
         Thread(target=self._start_heartbeat_tracker, name='HeartbeatTrackerThread', daemon=True).start()
 
     def _start_heartbeat_tracker(self):
-        self._hb_scheduler.enter(0, 0, self._disconnect_non_heartbeating_slaves)
+        self._hb_scheduler.enter(self._unresponsive_slaves_cleanup_interval, 0,
```
Comment (cmcginty) [inline app/master/cluster_master.py]: Just call `_disconnect_non_heartbeating_slaves()` to init the heartbeat tests

Comment (mayurva) [reply app/master/cluster_master.py]: Oh I see. Makes sense, but we want to wait for `_unresponsive_slaves_cleanup_interval` seconds after the master starts up. That way heartbeat doesn't run before the slaves get a chance to connect. Is there another way to achieve that?

Comment (cmcginty) [reply app/master/cluster_master.py]: Why wait? I don't think its worth the extra complexity in the code.

**Pull request closed without merging**